.github/workflows/elixir.yml

```diff
@@ -0,0 +1,29 @@
+name: Elixir CI
```

Comment:
Cool that this exists but I would like to stay on Travis CI for now and look into the Github workflows a little more before making this change.